src/app/pages/manage-organisation/manage-organisation-registration-step-1/manage-organisation-registration-step-1.component.html

```diff
@@ -47,12 +49,14 @@ <h1 class="govuk-heading-xl page-title">Create an account</h1>
         </svg>
       </button>
     </div>
-  </div>
-  <div class="govuk-grid-column-one-third">
-    <div class="card card-help">
-      <h2 class="card-title">Guides</h2>
-      <a href="https://www.crowncommercial.gov.uk/public-procurement-gateway-guides" class="govuk-link"
-        rel="noreferrer noopener" target="_blank">Learn about the Public Procurement Gateway, and how to set up an account.</a>
+    <div class="content-right">
```

Comment:
Not sure for using our own style here. is it possible to use the govuk grid to move the content to right